docling/cli/main.py

```diff
@@ -100,13 +101,20 @@ def export_documents(
             success_count += 1
             doc_filename = conv_res.input.file.stem
 
-            # Export Deep Search document JSON format:
+            # Export JSON format:
             if export_json:
                 fname = output_dir / f"{doc_filename}.json"
                 with fname.open("w", encoding="utf8") as fp:
                     _log.info(f"writing JSON output to {fname}")
                     fp.write(json.dumps(conv_res.document.export_to_dict()))
 
+            # Export HTML format:
+            if export_html:
+                fname = output_dir / f"{doc_filename}.html"
+                with fname.open("w", encoding="utf8") as fp:
+                    _log.info(f"writing HTML output to {fname}")
+                    fp.write(conv_res.document.export_to_html())
```

Comment:
Like this it will export HTML but not yet with embedded images. We need to further extend the CLI so people can control if they want the images.